projects/bootloader/src/config.c

```diff
@@ -54,8 +44,18 @@ StatusCode config_verify(void) {
                                    &s_config_2_blob, sizeof(BootloaderConfig), false));
   status_ok_or_return(persist_ctrl_periodic(&s_config_2_persist, false));
 
-  uint32_t config_1_check_crc = prv_compute_crc32(&s_config_1_blob);
-  uint32_t config_2_check_crc = prv_compute_crc32(&s_config_2_blob);
+  uint32_t config_1_check_crc = 0;
+  uint32_t config_2_check_crc = 0;
+
+  // Check for if the bootloader config is still zero. If it is not, then calculate the crc of the
+  // config to fill in the static configs
+  if (memcmp(&s_config_1_blob, &s_config_empty, sizeof(BootloaderConfig)) != 0) {
```

Comment:
I would say you can just declare `BootloaderConfig config_empty = { 0 }` here, so it's clear that you're just using it to compare. (also just a tip, if you're declaring something static, then you don't need to initialize it to zero, since static zeroes it by default)